### PR TITLE
Feat/add snyk code test support

### PR DESCRIPTION
**Pull request by ArturSnyk**
- [x] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
This introduces an **mpv** usage for snyk code.

#### Where should the reviewer start?
you should have the snykcode cli's ff,
and run it with `snyk code test` or `snyk code test <project_path>`

#### How should this be manually tested?
`snyk code test` or `snyk code test <project_path>`

#### Any background context you want to provide?
we will be adding more functionality around this flow, more error handling, analytics, and output functionality, later on

#### What are the relevant tickets?
https://snyksec.atlassian.net/browse/COD-123

#### Screenshots

image

### CODE REVIEW

**Review by robcresswell — CHANGES_REQUESTED** [review]
Just putting a blocker for ecosys arch to review

File: config.default.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "API": "https://snyk.io/api/v1",
   "devDeps": false,
   "PRUNE_DEPS_THRESHOLD": 40000,
-  "MAX_PATH_COUNT": 1500000
+  "MAX_PATH_COUNT": 1500000,
+  "CODE_CLIENT_PROXY_URL": "http://deeproxy.dev.snyk.io"
```
Comment (gitphill) [inline config.default.json:6]: shouldn't reference dev in this file? (I see API is pointing at production not dev)

**Review by JackuB — CHANGES_REQUESTED** [review]
There is still a vulnerability in coming from the `fast-glob`. We can't merge it if it's there

Screen Shot 2021-02-19 at 16 07 51

**Pull request closed without merging**